snapcraft/internal/lxd.py

```diff
@@ -204,18 +218,83 @@ def _ensure_container(self):
     def _setup_project(self):
         self._ensure_mount(self._project_folder, self._source)
 
+    def _get_container_address(self):
+        network = self._get_container_status()['state']['network']['eth0']
```

Comment:
Not an enormous deal, but this makes me a little nervous. We're using the default profile, right? What if the user changes that profile to configure their network devices differently? This device may not even exist, or may be an interface with no external network access.